Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,73 @@
+class Opentsdb < Formula
+  homepage "http://opentsdb.net"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.1.0/opentsdb-2.1.0.tar.gz"
+  sha256 "b76e36a2d744170dde0ecd7ff3b937cbe68dd9fab9ce885aab5847de3e975157"
+
+  def install
+    ENV.deparallelize
+    system "./build.sh --prefix=#{prefix}"
+    system "mkdir -p #{prefix}/share/opentsdb/static/gwt/standard/images/ie6"
+    Dir.chdir("build") do
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "true"
+  end
+
+  patch :p0, :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!